### PR TITLE
ONEAPP-3651 - Translations for Ecobee

### PR DESCRIPTION
**Pull request by juano2310**

### CODE REVIEW

File: smartapps/smartthings/ecobee-connect.src/i18n/ar-AE.properties [hunk 1]
```diff
@@ -18,3 +18,12 @@
 '''is connected to SmartThings'''={{deviceName}} متصل بـ SmartThings
 '''is disconnected from SmartThings, because the access credential changed or was lost. Please go to the Ecobee (Connect) SmartApp and re-enter your account login credentials.'''=تم قطع اتصال {{deviceName}} بـ SmartThings، لأن بيانات اعتماد الوصول قد تغيرت أو فُقدت. يُرجى الانتقال إلى التطبيق الذكي Ecobee (Connect)‎ وإعادة إدخال بيانات اعتماد تسجيل الدخول إلى حسابك.
 '''Your Ecobee thermostat '''=ثرموستات Ecobee
+'''Select your ecobee devices'''=تحديد أجهزة ecobee لديك
+'''Tap below to add or remove thermostats available in your ecobee account. Selected thermostats will connect to SmartThings.'''=انقر أدناه لإضافة أجهزة الثرموستات المتوفرة في حساب ecobee لديك أو إزالتها. سيتم توصيل أجهزة الثرموستات المحددة بـ SmartThings.
```
Comment (workingmonk) [inline smartapps/smartthings/ecobee-connect.src/i18n/ar-AE.properties:22]: Repeated below

File: smartapps/smartthings/ecobee-connect.src/i18n/ar-AE.properties [hunk 2]
```diff
@@ -18,3 +18,12 @@
...
+'''Log In'''=تسجيل الدخول
+'''Tap Next to continue to set up your ecobee thermostats.'''=انقر فوق ”التالي“ لمتابعة إعداد أجهزة الثرموستات ecobee.
+'''Your ecobee account is now connected to SmartThings!'''=حساب ecobee لديك متصل الآن بـ SmartThings!
```
Comment (workingmonk) [inline smartapps/smartthings/ecobee-connect.src/i18n/ar-AE.properties:25]: Repeat. already exists in the file on line 14

File: smartapps/smartthings/ecobee-connect.src/i18n/ar-AE.properties [hunk 3]
```diff
@@ -18,3 +18,12 @@
...
+'''Click 'Done' to finish setup.'''=انقر فوق ”تم“ لإنهاء الإعداد.
```
Comment (workingmonk) [inline smartapps/smartthings/ecobee-connect.src/i18n/ar-AE.properties:26]: repeated on line 15

**Comment by juano2310** [comment]
New PR: https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/3252

**Pull request closed without merging**